queries.sql

```diff
@@ -23,3 +23,57 @@ select * from animals where animal_name <> 'Gabumon';
 
 /*All animals with a weight between 10.4kg and 17.3kg*/
 select * from animals where weight_kg between 10.4 and 17.3;
+
+
+/* Transactions 1: Set species to 'unspecified' and Rollback*/
+begin;
+update animals set species = 'unspecified';
+
+rollback;
+
+/*Transaction 2: Set species based on name and commit*/
+begin;
+update animals set species = 'digimon' where animal_name like '%mon';
+update animals set species = 'pokemon' where species is null;
+commit;
+
+/*Transaction 3: Delete and Rollback*/
+begin;
+delete from animals;
+rollback;
+
```

Comment:
Great job so far. It would be great if you verify the transaction after `rollback` it. Currently, I can't see whether the data exists or not after the `rollback`. You can use ` Count` to check the data before and after `rollback`.
`DELETE FROM animals WHERE ;
 SELECT COUNT(*) FROM ANIMALS;` 
then `rollback` it and finally `SELECT COUNT(*) FROM ANIMALS;` to check the data is exist.

![image](https://github.com/iankiema/Vet-Clinic-Database/assets/101523582/47178974-12e4-4502-ba0f-26d8d3f05aac)